gubernator/view_build.py

```diff
@@ -34,6 +34,17 @@ def __init__(self):
         self.passed = []
         self.failed = []
 
+    def handle_suite(self, tree, filename):
+        for subelement in tree:
+            if subelement.tag == 'testsuite':
+                self.handle_suite(subelement, filename)
+            elif subelement.tag == 'testcase':
+                if 'name' in tree.attrib:
```

Comment:
this is required by most JUnit XSDs that I've seen. Do you have counter-examples?

Reply:
The tests, I guess

Reply:
I added this because the unit test cases were failing